### PR TITLE
Explicitly ask for ffmpeg logs

### PR DESCRIPTION
**Pull request by Artiume**
gimme yo ffmpeg logs, yo

**Changes**

**Issues**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/media_playback.md [hunk 1]
```diff
@@ -11,7 +11,10 @@ assignees: ''
 <!-- Use the Media Info tool (set to text format, download here: https://mediaarea.net/en/MediaInfo) or copy the info from the web ui for the file with the playback issue. -->
 
 **Logs**
-<!-- Please paste any log message from during the playback issue, for example the ffmpeg command line can be very useful. -->
+<!-- Please paste any log message from during the playback issue.-->
```
Comment (dkanada) [inline .github/ISSUE_TEMPLATE/media_playback.md]:
```suggestion
<!-- Please paste any log message from during the playback issue. -->
```

File: .github/ISSUE_TEMPLATE/media_playback.md [hunk 2]
```diff
@@ -11,7 +11,10 @@ assignees: ''
...
+
+**FFmpeg Logs**
+<!-- please paste any FFmpeg logs if remuxing or transcoding appears to be part of the issue. -->
```
Comment (dkanada) [inline .github/ISSUE_TEMPLATE/media_playback.md]:
```suggestion
<!-- Please paste any FFmpeg logs if remuxing or transcoding appears to be part of the issue. -->
```

File: .github/ISSUE_TEMPLATE/media_playback.md (hunk 1 above)
Comment (Artiume) [reply .github/ISSUE_TEMPLATE/media_playback.md]: those dang spaces lol

**Comment by dkanada** [comment]
@Artiume by the way, if you use the file view to apply suggestions you can commit them all at once with batch mode.

File: .github/ISSUE_TEMPLATE/media_playback.md [hunk 3]
```diff
@@ -11,7 +11,10 @@ assignees: ''
 <!-- Use the Media Info tool (set to text format, download here: https://mediaarea.net/en/MediaInfo) or copy the info from the web ui for the file with the playback issue. -->
 
 **Logs**
-<!-- Please paste any log message from during the playback issue, for example the ffmpeg command line can be very useful. -->
+<!-- Please paste any log message from during the playback issue. -->
```
Comment (dkanada) [inline .github/ISSUE_TEMPLATE/media_playback.md]:
```suggestion
<!-- Please paste any log messages from during the playback issue. -->
```

**Comment by Artiume** [comment]
Done. Ready to merge.

**Comment by Artiume** [comment]
Dang. I was too slow on my part. I want to add another line for Docker setup asking for Host_mode or Bridge mode.

**Pull request merged**